utils/spacewalk-common-channels.ini

```diff
@@ -2949,3 +2949,59 @@ gpgkey_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni
 gpgkey_id = %(_uyuni_gpgkey_id)s
 gpgkey_fingerprint = %(_uyuni_gpgkey_fingerprint)s
 repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/CentOS8-Uyuni-Client-Tools/CentOS_8/
+
+
+[rockylinux8]
+archs    = x86_64
+name     = RockyLinux 8 (%(arch)s)
+gpgkey_url = !!!!!!!!- Unable to locate
+gpgkey_id = !!!!!!- Unable to locate
```

Comment:
Keys can be found in the Rocky git repository for the time being but I have asked the Rocky team to make them available from download.rockylinux.org, which is the proper place. They will.

https://git.rockylinux.org/staging/rpms/rocky-release/-/tree/r8/SOURCES

Reply:
And added they are:
https://dl.rockylinux.org/pub/rocky/

Reply:
Thanks @paususe - amended commit with updated info.

url as given (official, not testing)
id from: gpg --import rocky.gpg
Fingerprint from: gpg --show-keys rocky.gpg

Reply:
Seems this was fixed (pending tests).